pkg/morph/client/notary.go

```diff
@@ -213,15 +258,26 @@ func (c *Client) UpdateNeoFSAlphabetList(list keys.PublicKeys) error {
 //
 // This function must be invoked with notary enabled otherwise it throws panic.
 func (c *Client) NotaryInvoke(contract util.Uint160, fee fixedn.Fixed8, method string, args ...interface{}) error {
+	if c.multiClient != nil {
+		return c.multiClient.iterateClients(func(c *Client) error {
+			return c.NotaryInvoke(contract, fee, method, args...)
+		})
+	}
+
 	if c.notary == nil {
 		return c.Invoke(contract, fee, method, args...)
 	}
 
 	return c.notaryInvoke(false, contract, method, args...)
 }
 
-func (c *Client) notaryInvokeAsCommittee(contract util.Uint160, method string, args ...interface{}) error {
-	return c.notaryInvoke(true, contract, method, args...)
+func (c *Client) notaryInvokeAsCommittee(method string, args ...interface{}) error {
+	designate, err := c.GetDesignateHash()
```

Comment:
Keep contract address in the `notaryInvokeAsCommittee` signature. The only usage of role management (designate) contract is just a case, not general rule.

Reply:
Ok, I'll return previous signature. I'd still implement the method with the proposed signature since two places of code to the same instructions. What would be the best name for it?